addons/payment_paypal/const.py

```diff
@@ -41,6 +41,6 @@
 # See https://developer.paypal.com/docs/api-basics/notifications/ipn/IPNandPDTVariables/
```

Comment:
There is no info about 'Cleared' status in the link. Maybe you could at least add the doc link about 'Cleared'  status to the dev notes of the task?

Reply:
I'm not sure we even have documentation about this; at least it should be a comment.